Vectors.Rmd

```diff
@@ -887,7 +884,7 @@ data.frame(
 knitr::include_graphics("diagrams/vectors/data-frame-list.png")
 ```
 
-List columns are easier to use with tibbles because you can provide them inside `tibble()`, and they are handled specially when printing:
+List columns are easier to use with tibbles because you can directly include them inside `tibble()`, and because printing them is handled automatically:
```

Comment:
```suggestion
List columns are easier to use with tibbles because you can directly include them inside `tibble()` and printing is handled automatically:
```

Reply:
"List columns are easier to use with tibbles because they can be directly included inside `tibble()` and they will be tidily printed:"

Reply:
printed tidily?